src/script/descriptor.cpp

```diff
@@ -137,6 +109,36 @@ std::string DescriptorChecksum(const Span<const char>& span)
     return ret;
 }
 
+
+namespace {
+
+////////////////////////////////////////////////////////////////////////////
+// Checksum                                                               //
```

Comment:
This comment shouldn't be moved.

Reply:
I'm not sure I understand. It's not quite clear in the diff, but the `// Checksum` comment was not moved. It remains after `namespace {`. The only comment that was moved is the one above `PolyMod`, describing the operation of that function, along with the function.

Reply:
In other words: The function does not need to move, only the `namespace {`

Reply:
Thanks, fixed in commit 6819fbd741891081b2679e4b763bb88dc997ffe7. Is there a way to re-run AppVeyor? Looks like it timed out.

Reply:
You can re-run the ci by squashing your commits according to https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#squashing-commits

Reply:
Ok, done in commit 0fc46f5b9262f016c42d2abe68d5125833f55416.